drivers/input/Kconfig

```diff
@@ -34,4 +34,9 @@ source "drivers/input/Kconfig.xpt2046"
 
 endmenu # Input Drivers
 
+config INPUT_MULTITOUCH_ENABLED
+	bool "Multitouch support enablement"
+	help
+	  Enables multitouch support.
```

Comment:
sorry I realized now this is in the generic file, just add it to the specific driver Kconfig, no need to make it subsystem wide and shows it up even if there's no touchscreen devices

Reply:
Agreed here- I would also namespace the Kconfig (IE `INPUT_GT911_MULTITOUCH_ENABLED`)

Reply:
Done!
That was the initial intention until I noticed some other touch screen drivers follow the same logic and might make use of the config when they start supporting multitouch too. I even thought it could be applied for any device that has the "lvgl-pointer" node but I understand where you come from.